packages/database/src/prisma-api.ts

```diff
@@ -116,6 +116,11 @@ export function getSdk(
               bulkCouponPurchases: true,
             },
           },
+          merchantPurchase: {
+            select: {
+              checkoutSessionId: true,
+            },
+          },
```

Comment:
this SDK function is what the invoice show page uses to look up the purchase, we lookup and include the `checkoutSessionId` in the result.